components/dashboard/debug.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License.AGPL.txt in the project root for license information.
+
+set -Eeuo pipefail
+source /workspace/gitpod/scripts/ws-deploy.sh deployment dashboard
```

Comment:
added this script as well which was useful in iterating on the dashboard component in a preview environment, the script is also present in `components/server` already